component/attr_cache/attr_cache.go

```diff
@@ -715,20 +615,29 @@ func (ac *AttrCache) RenameFile(options internal.RenameFileOptions) error {
 	err := ac.NextComponent().RenameFile(options)
 	if err == nil {
 		renameTime := time.Now()
+		ac.cacheLock.RLock()
+		defer ac.cacheLock.RUnlock()
+		// TODO: Can we just copy over the attributes from the source to the destination so we don't have to invalidate?
+
+		toBeDeleted, getErr := ac.cacheMap.get(options.Src)
```

Comment:
This variable name is confusing. Yes, this entry gets deleted, but not here.
Something generic like `sourceItem` would work fine.